owc-backend/package.json

```
@@ -7,7 +7,7 @@
   "author": "louicoder <musanje2010@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "dev": "nodemon index.js"
+    "dev": "dev index.js"
```

@davidofug I'm not sure this command will run because dev will be undefined, in other words it's not installed. I had put nodemon for dev script because it helps restart the server every time you make a change without doing it manually
```suggestion
    "dev": "nodemon index.js"
```
```suggestion
    "start": "node index.js"
```

---

Thank you for the gotcha! Fixed.